apps/testing/src/main/java/com/akto/test_editor/Utils.java

```diff
@@ -576,4 +577,44 @@ public static boolean evaluateResult(String operation, boolean currentRes, boole
         return currentRes || newVal;
     }
 
+    public static String convertToHarPayload(String message, int akto_account_id, int time, String type, String source) throws Exception {
+
+        Map<String, Object> json = gson.fromJson(message, Map.class);
+
+        Map<String, Object> req = (Map) json.get("request");
+        Map<String, Object> resp = (Map) json.get("response");
+
+        Map<String, Object> reqHeaders = new HashMap<>();
+        try {
+            reqHeaders = mapper.readValue((String) req.get("headers"), HashMap.class);
+            reqHeaders.remove("x-akto-ignore");
+        } catch (Exception e) {
+            // TODO: handle exception
+        }
+
+        String requestHeaders = mapper.writeValueAsString(reqHeaders);
+        String responseHeaders = (String) resp.get("headers");
+
+        String path = OriginalHttpRequest.getFullUrlWithParams((String) req.get("url"), (String) req.get("queryParams"));
+        String contentType = (String) reqHeaders.get("content-type");
+
+        Map<String,String> result = new HashMap<>();
+        result.put("akto_account_id", akto_account_id+"");
```

Comment:
Use this instead https://github.com/akto-api-security/akto/blob/master/libs/utils/src/main/java/com/akto/open_api/parser/Parser.java#L309

Reply:
the constants are not properly defined. Let's discuss this later and modify, for now ignoring this comment